mainWidget/main_widget_FloorsAndRooms.yaml

```diff
@@ -1,139 +1,219 @@
+                                            large: 50
+                                            medium: 50
+                                            small: 90
+                                            style:
+                                              overflow: hidden
+                                            width: 100
+                                            xlarge: 33
+                                            xsmall: 100
+                                          slots:
+                                            default:
+                                              - component: widget:main_widget_Light_Card
+                                                config:
+                                                  Title: "=loop.lightItem.metadata.uiSemantics ? loop.lightItem.metadata.uiSemantics.config.equipment + loop.lightItem.metadata.uiSemantics.config.preposition + loop.lightItem.metadata.uiSemantics.config.location : loop.lightItem.label"
+                                                  groupItem: =loop.lightItem.name
+                                                  groupItemLabel: =loop.lightItem.label
+                                                  groupItemLocation: =loop.rooms.label
+                                                  powerItem: = loop.lightPowerItem.name
                             - component: oh-repeater
                               config:
-                                fetchMetadata: semantics,metadata
-                                filter: loop.modeItem.type=="Switch"
-                                for: modeItem
-                                groupItem: =loop.equipmentItem.name
-                                sourceType: itemsInGroup
+                                fetchMetadata: semantics,metadata,listWidget
+                                filter: (loop.equipmentItem.metadata.semantics.config) && (loop.equipmentItem.metadata.semantics.config.hasLocation) && (loop.equipmentItem.metadata.semantics.config.hasLocation == loop.rooms.name)
+                                for: equipmentItem
+                                fragment: true
+                                itemTags: Blinds
+                                sourceType: itemsWithTags
                               slots:
                                 default:
-                                  - component: widget:main_widget_Rollershutter_Card
+                                  - component: oh-repeater
                                     config:
+                                      fetchMetadata: semantics,metadata,uiSemantics
+                                      filter: (loop.shutterItem.type == "Rollershutter") && (loop.shutterItem.tags).includes("Control")
```

Comment:
Whay adding the tag to the filter. Item needs to be of type Rollershutter, no other filter needed.

Reply:
As described in the PR: Filter for blinds extended. There are users who have HmIP in use and have to use proxy items. These are then not tagged but included in the group.

So in the group, as a proxy item, there is another Rollershutter item.

More details needed with item code?

Reply:
Ok, understood, so we need to add this to the readme, that the model needs to be extended. For me, without the Control tag, it did not show the Cards.
But still don‘t understand the need of a proxy item, but that’s a different story.

Reply:
Ok